keyboards/jpe230/big_knob/config.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 jpe230 (@jpe230)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define ENCODER_RESOLUTION 4
+
+// SPI pins
+#define SPI_DRIVER SPID0
+#define SPI_SCK_PIN GP22
+#define SPI_MOSI_PIN GP23
+#define SPI_MISO_PIN GP20 // Unused
```

Comment:
Could be `NO_PIN` on develop already

Reply:
Sadly I'm targeting master :( but will update once develop is merged with master